frontend/src/Editor/Viewer/PreviewSettings.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import Icon from '@/_ui/Icon/solidIcons/index';
+import { OverlayTrigger } from 'react-bootstrap';
+import { useAppVersionStore } from '@/_stores/appVersionStore';
+import { useAppInfo } from '@/_stores/appDataStore';
+import { AppVersionsManager } from '../AppVersionsManager/AppVersionsManager';
```

Comment:
If possible, can we use `@` to import the components?

Reply:
Done